### DISCUSSION TITLE
does cpu limits (pl1+pl2) works for anyone?

### DISCUSSION THREAD

**Discussion by cosmicAP** [seerge/g-helper#1138] [Q&A]
my laptop asus tuf dash f15 ( i5 12450h and rtx 3050) runs very hot 95 flat so i tried the cpu power limit and it doesn't seem to work my cpu still stuck at 95 , does anyone have a solution or is facing similar issues? does it requires custom fan control?

**Comment by seerge** [comment]
Some models want a custom fan to be applied as well, does doing that change the situation?

If that still doesn't have effect, some rare devices may require a special "manual" mode to set limits. To test if it's needed for you you need to add following line to ``config.json`` of the app (and start app after that again)

```
"manual_mode": 1,
```

**Reply by cosmicAP** [reply]
hello seerge , apologize for the late response , here is the log file

[log.txt](https://github.com/seerge/g-helper/files/12446727/log.txt)


what i have noticed is only after reaching at last 5 watt that it starts to show any difference. my laptop lags very bad at cpu watt 5

should i be able to give that much low watt to cpu , my cpu i5 12450h has minimum wattage of 35w . this is after custom fan setting

**Reply by seerge** [reply]
Then it means that power limits work for you as well

**Reply by cosmicAP** [reply]
well i can't tell for sure but i try to do more research . I will test this on plugged in , thanks for the response seerge as always your app rocks

**Reply by seerge** [reply]
The easiest way to test is to monitor it via HWinfo and run Cinebench23 loading CPU on max. Plugged in of course, as otherwise it can be limited in some strange by bios cause of battery restrictions

**Reply by cosmicAP** [reply]
thanks for the suggestion i will give it a try if i caught anything i will notify you.

**Comment by gnartS** [comment]
Ive got a G16 2023 and although these modes work, i have to tick "Apply Custom Fan Curve" as well as "Apply Power Limits" for them to start working,

If i dont tick them, the siders do nothing.

Summary;-
To use a power limit i have to (i) drag slider (ii) tick apply power limits and (iii) tick apply custom fan curve even if using default curves.

Is this expected behaviour or a bug?

**Reply by seerge** [reply]
@gnartS this is specifics of some model's BIOS (as in armoury manual mode always sets all together). I can add your model to the list where app will silently apply fan curve as well, next to power limits

**Comment by gnartS** [comment]
OK thats good to know im doing it correctly.

Exact model is GU603VI if that matters (G16 2023, RTX 4070, i9).  Bios 313.
